### PR TITLE
Refactor duplicate CSS rules

### PR DESCRIPTION
**Pull request by RMartin0717**
Refactor CSS to remove duplicate properties across media query breakpoints

Screen Shot 2021-04-11 at 3 12 09 PM
Screen Shot 2021-04-11 at 3 11 59 PM
Screen Shot 2021-04-11 at 3 11 16 PM

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -636,91 +596,35 @@ p {
...
   .link-box {
-    /* padding: 1vw; */
     margin-top: .75vw;
-    /* background-color: #EFEFEF; */
   }
-
-
 }
 
 @media only screen and (min-width: 920px) {
```
Comment (mcraig2342) [inline styles.css:616]: NIce job using 2 media queries for multiple breakpoints for tablet and phone!

File: styles.css [hunk 2]
```diff
@@ -348,16 +348,11 @@ template {
   display: none;
 }
```
Comment (mcraig2342) [inline styles.css:350]: Good work formatting this across multiple browsers, I like it!

File: styles.css [hunk 3]
```diff
@@ -454,17 +435,14 @@ p {
...
   display: flex;
   justify-content: space-between;
-
   width: 100%;
   height: 100%;
   background-color: #2B2B2B;
-
 }
 
 .arrow-button {
```
Comment (mcraig2342) [inline styles.css:448]: These look good in your screenshots, but when I open it on my desktop after cloning down your repo it looks like this (if I can add screenshots in here, if not i will send you the image):

Screen Shot 2021-04-11 at 3 28 06 PM

File: styles.css [hunk 4]
```diff
@@ -636,91 +596,35 @@ p {
...
-    display: flex;
-    align-items: center;
-    justify-content: center;
-
-  }
-  .card-body {
-    /* height: 15vh; */
-  }
-
-
```
Comment (mcraig2342) [inline styles.css:725]: good job removing properties that you didn't change in your media queries

File: styles.css [hunk 5]
```diff
@@ -636,91 +596,35 @@ p {
...
   .card {
-    display: flex;
-    flex-direction: column;
     width: 23vw;
     height: 35vw;
   }
 
-
   .card-section {
-    display: grid;
```
Comment (mcraig2342) [inline styles.css:701]: nice user of grid!

**Review by mcraig2342 — COMMENTED** [review]
Great job overall! I like your use of grid, media queries and normalization across browsers. Maybe look into why those images might not be showing up on my end, not sure why that would be happening or if i'm doing something wrong lol.

File: styles.css (hunk 3 above)
Comment (kip-west) [reply styles.css:448]: Same for me! In the dev tools, it looks like there are some 403 errors coming back on anything from Flaticon. Any time we load an image/icon from a link, we're subject to these sort of pitfalls, where something is blocked from loading or the host domain changes. To account for this, I'd start downloading all of your images/icons and adding them into your repo, and then importing them across your files!

Screen Shot 2021-04-11 at 5 56 26 PM

**Review by kip-west — COMMENTED** [review]
Great work with your static comp! I like how thorough you are when formatting across different browsers- very thoughtful. I like your styling choices- super clean and your `box-shadow` makes the cards jump off the page. Great refactor here, it seems like you were able to clean up some redundant CSS!

One potential change you might consider is turning your `card-section` into a scrollable container! I'm attaching a gif here showing how a white background peeps at the top of the screen if you scroll too far up. Check out the [overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) property in CSS! I add this to most containers that have varying content (whether it be cards, text elements, or other html tags) to help the page stay responsive.
scroll-RM

Another thing to consider when you think about the height of any containers (such as your header or card container) is to incorporate `max-height` or `min-height`! When  someone is viewing your site on a really wide screen (see absurd screenshot below), the header takes up a large amount of the height. If you use max and min values for these heights, it creates bounds for your header to grow/shrink as needed with the device, without getting out of control!
Screen Shot 2021-04-11 at 6 32 56 PM

File: styles.css [hunk 6]
```diff
@@ -406,20 +393,14 @@ h1 {
```
Comment (kip-west) [inline styles.css:393]: It looks like the `h1` color styling conflicts with your background styling! The text `Shares` disappears... 

From my understanding,`h1` tags have an effect on [SEO](https://www.greenlanemarketing.com/resources/articles/seo-101-seo-and-multiple-h1-tags/) & [screen reader interpretation](https://accessibility.psu.edu/headingshtml/). There is a school of thought around only having one `h1` on your page, so you might consider making `Shares` a stylized `h2` with your darker font color.

File: styles.css [hunk 7]
```diff
@@ -552,32 +530,25 @@ p {
```
Comment (kip-west) [inline styles.css:530]: Great work handling responsive design with multiple breakpoints!! You've really nicely covered a wide range of screen sizes here.

I would even consider adding a few more breakpoints to smooth out the transitions as you grow and shrink your page. Below you'll find a screenshot with one template for breakpoints! These are used in [Bootstrap](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints), which is a styling package that might be cool to explore in Mod 4 as stretch tech. In addition to dragging the responsive window width, I also like using the dev tools dropdown for common devices to get a sense for UX on an iPad vs. iPhone, when their device is in landscape vs. portrait, etc.
Screen Shot 2021-04-11 at 6 17 43 PM

Note here how there is a gap between your two rows of cards! I like Bootstraps suggested breakpoints as it helps get into all the weird "in between" screen sizes.
Screen Shot 2021-04-11 at 6 21 23 PM

**Pull request merged**